Panels/Viewport.js

```diff
@@ -138,7 +138,9 @@ const ViewportBase = class extends Component {
 
 	removeTransitioningClass = () => {
 		if (this.node) {
-			this.node.classList.remove(css.transitioning);
+			setTimeout(() => {
```

Comment:
I saw the flickering but could you explain why?
🤔 Is there any other way instead of `setTimeout`?

Reply:
I checked again and I did not see the flickering, and `setTimeout` was used to prevent it. Can you please verify again or help me reproduce this behavior?

Reply:
I meant, if I removed `setTimeout`, I can see the flickering.
What if we remove `setTimeout` here and add `overflow: hidden` property in the `.panels.partial` from `Panels.module.less`?
I think we don't need to draw outside of Panels if it covers partial.
What do you think? Do you have any conerns?

Reply:
I checked, and it looks good. Thank you!